### DISCUSSION TITLE
Flutter web support

### DISCUSSION THREAD

**Discussion by andre-paraense** [oddbit/flutter_facebook_app_events#139] [Ideas]
---
name: Feature request
about: Flutter web support
title: ''
labels: ''
assignees: ''

---

**Is your feature request related to a problem? Please describe.**
Currently, there is no support for Flutter web.

**Describe the solution you'd like**
Provide support for web as well, linking to the web javascript Facebook SDK

**Additional context**
This would be very useful for anyone building progressive web apps with Flutter and wanting to drive traffic through Facebook Ads.

**Comment by DennisAlund** [comment]
Currently, we don't have resource on the team for anyone to look into this. Would be happy to discuss and plan together with someone in the community who would like to give it a shot 🙏

**Reply by andre-paraense** [reply]
I will need it in one of my projects soon, depending on the priority I might take it on

**Reply by DennisAlund** [reply]
I would be willing and happy to help and support if you get a good start on it. My main blocker is that I haven't developed any projects: flutter for web yet, so I am unfamiliar with the foundation that we would need to set up to get it started. 

But if you have a good foundational start I would be able to support with the actual implementation and if you are having any questions on how to apply the bridging of methods.
